templates/docs/layouts/documentation.md

```diff
@@ -0,0 +1,86 @@
+---
+wrapper_template: '_layouts/docs.html'
+context:
+  title: Documentation | Layouts
+---
+
+## Documentation layout
+
+<hr>
+
+### Structure
+
+The documentation page layout can be built using the Vanilla grid classes and common components. It consists of 3 horizontal areas that span the entire fixed width of the grid: header, content, footer.
+
+![Documentation layout structure](https://assets.ubuntu.com/v1/cdf09557-Documentation+layout%401x.svg)
+
+At the large breakpoint, the content area is further divided into an aside (3 columns) and a main content area (9 columns).
+
+At smaller breakpoints, the aside is moved offscreen and shown / hidden using a toggle.
+
+#### Header
+
+The header with the main navigation, built with the [navigation component](/docs/patterns/navigation#global-navigation).
```

Comment:
How about:

"Place the main navigation and any other full width elements in the header. This could include a strip with a search, a hero element, etc."